### DISCUSSION TITLE
Can't force 120hz on external monitor with dex on Galaxy Fold 6

### DISCUSSION THREAD

**Discussion by Ilkygul** [tribalfs/GalaxyMaxHzPub#225] [Q&A]
When using Samsung dex with 144hz 2k external usb c monitor I can't force 120 hz with the Galaxy Max Hz app. I tried it on the Galaxy Fold 5 and 6, but without result. It is capped at 60 hz

**Comment by tribalfs** [comment]
This is beyond scope of the app. The app is intended for the device's default display only.
